src/helpers/helpers.canvas.js

```diff
@@ -47,7 +47,7 @@ var exports = module.exports = {
 	drawPoint: function(ctx, style, radius, x, y) {
 		var type, edgeLength, xOffset, yOffset, height, size;
 
-		if (typeof style === 'object') {
+		if (style && typeof style === 'object') {
```

Comment:
`if (helpers.isObject(style))`

Reply:
Actually, `helpers.isObject` doesn't work in this case since it returns `true` only for plain object.